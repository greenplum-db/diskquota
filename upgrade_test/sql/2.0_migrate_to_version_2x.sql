\! gpconfig -c shared_preload_libraries -v 'diskquota-2.0.so' > /dev/null
\! gpstop -raf > /dev/null

\! gpconfig -s 'shared_preload_libraries'

\c
alter extension diskquota update to '2.0';
-- FIXME add version check here
