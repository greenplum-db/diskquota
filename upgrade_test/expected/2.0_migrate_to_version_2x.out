\! gpconfig -c shared_preload_libraries -v 'diskquota-2.0.so' > /dev/null
\! gpstop -raf > /dev/null
\! gpconfig -s 'shared_preload_libraries'
Values on all segments are consistent
GUC          : shared_preload_libraries
Master  value: diskquota-2.0.so
Segment value: diskquota-2.0.so
\c
alter extension diskquota update to '2.0';
-- FIXME add version check here
